### ISSUE TITLE
Impossible to access an attribute ("startDate") on a null variable in "__string_template__

### ISSUE THREAD

**Issue by tommysvr** [unionco/calendarize#44]
I'm getting this very strange error on pages that don't relate to Calendarize ever since I installed it.

The full error is:
```Twig Runtime Error – Twig\Error\RuntimeError

Impossible to access an attribute ("startDate") on a null variable in "__string_template__72438934248b0d954161cc5bb0a7c4887491e74869a6418409c7eb1144d3966b" at line 14.
```

I receive it when trying to save entries that don't have Calendarize fields; when I do add them, I am able to save the entry without issue. It also occurs when editing a Section itself, despite not having anything to do with Calendarize (I think?)

Hope someone can help with this as it's really bugging me! :)

**Comment by fvaldes33** [comment]
@tommysvr - What version of craft and plugin? Also could you provide steps to reproduce. Thanks

**Comment by tommysvr** [comment]
Thanks for the reply. I'm running Craft Pro 3.3.20.1 and Calendarize 1.2.16. It's hard to state the exact steps I took, however, I feel like it happened after I deleted the eventDetails field randomly. I understand that's extremely vague, but I'm really not sure what else I could have done to make it happen.

The other thing is, when I push my code and db to the live server, everything works fine. So this obviously means something with my local dev server. But it's also clearly referencing a Calendarize field. I've tried empty cache, remove and reinstall plugins (Calendarize and others) but nothing has worked so far and I'm completely lost.
